lib/spatial_features/importers/geojson.rb

```diff
@@ -0,0 +1,29 @@
+require 'ostruct'
+require 'digest/md5'
+
+module SpatialFeatures
+  module Importers
+    class GeoJSON < Base
+      def cache_key
+        @cache_key ||= Digest::MD5.hexdigest(features.to_json)
```

Comment:
I think a more performant cache key would be to simply hexdigest the incoming JSON, not convert it to feature objects and then join those.

Reply:
Great point 👍